Backend/WebApi-PointMap/ErrorHandling/AuthorizationErrorHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using DataAccessLayer.Database;

```

Comment:
Check with vong, to make sure this is okay way of being done. Little confusing, don't know if easier way exists